src/Hero6.DesktopGL/Hero6.DesktopGL.csproj

```diff
@@ -15,6 +15,7 @@
     <MonoGameContentBuilderExe Condition=" '$(OS)' != 'Windows_NT' And Exists ('/opt/monogame-pipeline/MGCB.exe') ">/opt/monogame-pipeline/MGCB.exe</MonoGameContentBuilderExe>
     <TargetFrameworkVersion>v4.5.1</TargetFrameworkVersion>
     <TargetFrameworkProfile />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
I can't help but feel, that if we're going to add this property, we should add them in other projects as well.